### ISSUE TITLE
Too much recursion

### ISSUE THREAD

**Issue by timhwang21** [avocode/avocode#2227]
---
* **Version**: 3.6.2-web.1
* **Web/Desktop**: Web
* **Platform**: macosx 10.13
* **Browser**: firefox 62.0
* **Avocode UID**: 240986
---
```
too much recursion
escape@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:58:4307
l@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:58:363921
e@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:58:364165
e.exports@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:58:364004
e.exports@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:516568
_reconcilerUpdateChildren@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:505269
_updateChildren@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:506206
updateChildren@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:506104
_updateDOMChildren@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:497610
updateComponent@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:495816
receiveComponent@https://cdn.app.avocode.com/dist/v-65d213b/js/avocode-web.min.js:75:495369
receiveComponent@h
```

**Comment by crashmajstr** [comment]
Hello @timhwang21 ,
would you be able to describe when exactly did this error happen? Do you remember what were you doing when the error popped out?

Were you trying to open a specific design? If so, does the error pop up everytime you try to open the same design?

**Comment by timhwang21** [comment]
Sorry if I'm not helpful but I dont think I was doing anything dramatically different. However I was seeing errors on literally everything I did while I had a Google Meet video chat on with 10+ participants. Once I left the meeting this error stopped as well.

**Comment by crashmajstr** [comment]
@timhwang21 it's ok, I reached out to the dev team and it's a known issue. We should be able to release the fix during the next week! Sorry for the inconvenience.

**Comment by timhwang21** [comment]
Thanks! Out of curiosity does it in fact have an interaction with other resource-heavy things going on in the browser like webchat? Or was it unrelated?

**Comment by crashmajstr** [comment]
I guess it's unrelated, but can't be 100% sure about that.

**Comment by vojtatranta** [comment]
@timhwang21 it is unrelated and it will be fixed in the next release.

**Comment by vojtatranta** [comment]
@timhwang21 this should be fixed in `3.6.2-web.3` - currently deployed on the web.
